src/components/profile/PurchasedProjectItem.tsx

```
@@ -0,0 +1,172 @@
+            </button>
+
+            {/* 판매 완료 오버레이 */}
+            {isCompleted && (
+              <div
+                className="absolute inset-0 flex flex-col items-center justify-center text-white cursor-pointer"
+                onClick={handleProjectClick}
+              >
+                <div className="bg-white rounded-full p-2 mb-2">
+                  <Check className="h-8 w-8 text-sub-gray" />
+                </div>
+                <div className="text-xl font-bold">판매 종료</div>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)